src/main/java/model/Amount.java

```diff
@@ -0,0 +1,25 @@
+package model;
+
+public class Amount {
+    public static final int LOTTO_UNIT_PRICE = 1000;
+    private final int cost;
```

Comment:
상수와 상태 값은 구분지어 주는 것이 좋습니다 :)
```suggestion
    public static final int LOTTO_UNIT_PRICE = 1000;
    
    private final int cost;
```